### PR TITLE
Add Hebrew translation

### PR DESCRIPTION
**Pull request by avma**
Add he.po file

### CODE REVIEW

**Comment by bk138** [comment]
Thanks so much for the contribution! You're almost there, the PR is only missing the build system integration, see https://github.com/bk138/gromit-mpx/blob/master/po/README.md and for how it was done before https://github.com/bk138/gromit-mpx/commits/master/po

**Review by bk138 — CHANGES_REQUESTED** [review]
CMake integration still to be done.

**Comment by avma** [comment]
Hope this time I did better...

File: CMakeLists.txt [hunk 1]
```diff
@@ -90,11 +91,12 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/de.gmo DESTINATION ${CMAKE_INSTALL_DAT
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/es.gmo DESTINATION ${CMAKE_INSTALL_DATADIR}/locale/es/LC_MESSAGES/ RENAME ${LOCALE_DOMAIN}.mo)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/it.gmo DESTINATION ${CMAKE_INSTALL_DATADIR}/locale/it/LC_MESSAGES/ RENAME ${LOCALE_DOMAIN}.mo)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/fa.gmo DESTINATION ${CMAKE_INSTALL_DATADIR}/locale/fa/LC_MESSAGES/ RENAME ${LOCALE_DOMAIN}.mo)
-
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/he.gmo DESTINATION ${CMAKE_INSTALL_DATADIR}/locale/he/LC_MESSAGES/ RENAME ${LOCALE_DOMAIN}.mo)
```
Comment (bk138) [inline CMakeLists.txt:94]: please leave the newline before the configure_file stanza in there :-)

File: CMakeLists.txt [hunk 2]
```diff
@@ -90,11 +91,12 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/de.gmo DESTINATION ${CMAKE_INSTALL_DAT
...
 configure_file(
     "${CMAKE_CURRENT_SOURCE_DIR}/cmake_uninstall.cmake.in"
     "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
     IMMEDIATE @ONLY)
 
 add_custom_target(uninstall
     COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
+
```
Comment (bk138) [inline CMakeLists.txt:102]: Please remove that unrelated newline again.

**Review by bk138 — CHANGES_REQUESTED** [review]
Thanks so much! Only some really minor cosmetic things left!

**Pull request merged**